.ci/chatops-commands/ci-problems-analyze.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+if [ -z "$1" ]; then
+    >&2 echo "Usage: $0 PR_NUMBER"
+    exit 1
+fi
+
+set -e
+
+OWNER_NAME="camunda"
+REPO_NAME="camunda"
+PR_NUMBER="$1"
+PR_HEAD_SHA=$(gh api -X GET "repos/$OWNER_NAME/$REPO_NAME/pulls/$PR_NUMBER" --jq '.head.sha')
+
+echo "## 🔎 GHA problems for PR #$PR_NUMBER"
+echo ""
+echo "For most recent commit $PR_HEAD_SHA:"
+echo ""
+
+gh api -X GET "repos/$OWNER_NAME/$REPO_NAME/actions/runs?sha=$PR_HEAD_SHA" --jq '.workflow_runs[]' | while read -r workflow_run; do
+  workflow_run_id=$(echo "$workflow_run" | jq -r '.id')
```

Comment:
My plan to catch errors is by exiting the Bash script when any command returns a non-zero exit code. This is achieved by the `set -e` at the beginning. This protects against failing GitHub API requests since the `gh` tool then exists non-zero.

For `jq` this also happens correctly as I tested locally (exit code 4):

```
$ echo "some error message from previous command" | jq -r '.id'
parse error: Invalid numeric literal at line 1, column 5
```

Reply:
Ah you are right! I hadnt enabled `set -e` when testing